platformio.ini

```diff
@@ -12,19 +12,25 @@
 src_dir = firmware/esp32_ttgo
 data_dir = web/static
 
-[env:ttgo-lora32-v1]
+[env]
 platform = espressif32
 board = ttgo-lora32-v1
 framework = arduino
-upload_port = /dev/ttyUSB0
-monitor_port = /dev/ttyUSB0
+upload_port = /dev/cu.SLAB_USBtoUART
```

Comment:
`/dev/cu.SLAB_USBtoUART` doesn't work for me. I have to switch to `/dev/ttyUSB0`. Not sure which is more common, maybe we default to `/dev/ttyUSB0` and leave `/dev/cu.SLAB_USBtoUART` commented out, or vice versa? I'm sure platformio has ways of handling multiple upload port options.

Reply:
Yeah that's completely platform dependent, I'm using macOS. I can leave it as `ttyUSB0`